pkg/config/remote/client.go

```diff
@@ -63,16 +63,36 @@ type Client struct {
 
 // NewClient creates a new client
 func NewClient(agentName string, agentVersion string, products []data.Product, pollInterval time.Duration) (*Client, error) {
+	return newClient(agentName, true, agentVersion, products, pollInterval)
+}
+
+// NewUnverifiedClient creates a new client that does not perform any TUF verification
+func NewUnverifiedClient(agentName string, agentVersion string, products []data.Product, pollInterval time.Duration) (*Client, error) {
+	return newClient(agentName, false, agentVersion, products, pollInterval)
+}
+
+func newClient(agentName string, doTufVerification bool, agentVersion string, products []data.Product, pollInterval time.Duration) (*Client, error) {
 	ctx, close := context.WithCancel(context.Background())
 	grpcClient, err := grpc.GetDDAgentSecureClient(ctx)
 	if err != nil {
 		close()
 		return nil, err
 	}
-	repository, err := state.NewRepository(meta.RootsDirector().Last())
-	if err != nil {
-		close()
-		return nil, err
+
+	var repository *state.Repository
+
+	if doTufVerification {
+		repository, err = state.NewRepository(meta.RootsDirector().Last())
+		if err != nil {
+			close()
+			return nil, err
+		}
+	} else {
+		repository, err = state.NewUnverifiedRepository()
+		if err != nil {
+			close()
+			return nil, err
+		}
 	}
```

Comment:
Since this returns on error, the error check could be taken out of the conditional statement to avoid duplication and save some readability.

```suggestion
	if doTufVerification {
		repository, err = state.NewRepository(meta.RootsDirector().Last())
	} else {
		repository, err = state.NewUnverifiedRepository()
	}
	if err != nil {
	    close()
		return nil, err
	}
```